exporter/awsprometheusremotewriteexporter/auth.go

```diff
@@ -158,20 +153,3 @@ func cloneRequest(r *http.Request) *http.Request {
 	}
 	return r2
 }
-
-// addToUserAgent will format the provided name and version to the name/version format.
-// If the extra parameters are provided they will be added as metadata to the
-// name/version pair resulting in the following format.
-// "name/version (extra0; extra1; ...)"
-// It then adds the string to the end of the request's current User-Agent.
-func addToUserAgent(req *http.Request, name, version string, extra ...string) {
-	ua := fmt.Sprintf("%s/%s", name, version)
-	if len(extra) > 0 {
-		ua = fmt.Sprintf("%s (%s)", ua, strings.Join(extra, "; "))
-	}
-	curUA := req.Header.Get("User-Agent")
```

Comment:
This logic checking if there is a user agent or not seems like we should still have in the new version

Reply:
I have added this checking